jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -489,43 +489,46 @@ Browser: ${navigator.userAgent}`
     }
 
     const entityId = state.postDeploymentPanel.entityId;
-    const nowMillis = new Date().getTime();
+    // If there are no new entites then there's no need to display the panel.
```

Comment:
entities

Reply:
Why wouldn't there be an entity? Can you add a comment to this effect, or maybe it's another bug?

Reply:
I presume there are no entities any time you're not deploying new applications. At the least in this case my model only contained a machine.